packages-exp/auth-exp/test/integration/flows/phone.test.ts

```diff
@@ -69,7 +70,11 @@ describe('Integration test: phone auth', () => {
     auth = getTestInstance();
     fakeRecaptchaContainer = document.createElement('div');
     document.body.appendChild(fakeRecaptchaContainer);
-    verifier = new RecaptchaVerifier(fakeRecaptchaContainer, undefined, auth);
+    verifier = new RecaptchaVerifier(
+      fakeRecaptchaContainer,
+      undefined as any,
```

Comment:
Has to cast to `any` to workaround TS error here and below. `RecaptchaVerifier` doesn't take `undefined` as the second argument, but I guess it's okay if the test doesn't touch the second argument.